roles/pulibrary.solr/tasks/main.yml

```diff
@@ -26,12 +26,13 @@
   when: solr_install_path_status.stat.isdir is not defined
   register: solr_download_status
 
-- name: Expand Solr.  # noqa 503
+- name: Expand Solr.  # noqa 503 ANSIBLE0016
```

Comment:
what is this?

Reply:
I added the `.ansible-lint` which is the pattern used in other roles @hackmastera